components/Input/Input.jsx

```diff
@@ -49,40 +41,77 @@ const InputTag = styled.input`
     display: none;
   }
 
-  &:focus {
-    border-color: ${Colors.PRIMARY['500']};
+  &:focus + ${InputLabel} {
+    color: ${ColorsDeprecated.PRIMARY['500']};
   }
 
-  ${props =>
-    props.error &&
+  ${({ placeholder, defaultValue }) =>
+    placeholder &&
+    !defaultValue &&
     `
-    border-color: ${Colors.DANGER['500']};
+    color: ${Colors.BLACK['400']};
   `}
 
-  &:focus + ${InputLabel} {
-    color: ${Colors.PRIMARY['500']};
-    ${sharedStyle}
+  :-webkit-autofill {
+    box-shadow: 0 0 0px 1000px ${Colors.BLUE['200']} inset;
+    -webkit-text-fill-color: ${Colors.BLACK['700']};
   }
 `;
 
 const InputIcon = styled(Icon)`
-  position: absolute;
-  right: 2px;
   cursor: pointer;
+  position: absolute;
+  right: 12px;
+  top: 50px;
+
+  ${({ description }) =>
+    description &&
+    `
+    top: 70px;
+  `};
 `;
 
-const InputFieldGroup = styled(FieldGroup)`
-  margin: 40px 0 20px;
+const InputSearchIcon = styled(InputIcon)`
+  left: 12px;
+`;
 
-  &:first-child {
-    margin-top: 20px;
-  }
+const InputErrorIcon = styled(InputIcon)`
+  color: ${Colors.ERROR['500']};
 `;
 
 const InputErrorMessage = styled(ErrorMessage)`
-  margin-top: 8px;
+  padding: 8px 12px;
+`;
+
+const HelperText = styled.span`
+  color: ${Colors.BLACK['700']};
```

Comment:
In our conversations, we specified that the labels should inherit color from body, is that right?